README.md

```diff
@@ -57,7 +57,7 @@ resolver | true | - | The method to wrap, and how to interpret the returned data
 responseParser | false | (system) | The method that format the results from the resolver into an indexed collection. Accepts indexed collections or arrays of objects with an `id` property. Uses the format `<function(response, requestedIds, params)>`
 uniqueParams | false | `[]` | The list of parameters that, when passed, generate unique results. Ex: 'language', 'view', 'fields', 'country'. These will generate different combinations of cache keys.
 timeout | false | `null` | The maximum time allowed for the resolver to resolve.
-cache | false | <pre>{&#13;&#10;&nbsp;&nbsp;limit: 60000,&#13;&#10;&nbsp;&nbsp;ttl: 60000&#13;&#10;}</pre> | Caching options for the data
+cache | false | <pre>{&#13;&#10;&nbsp;&nbsp;limit: 5000,&#13;&#10;&nbsp;&nbsp;ttl: 300000&#13;&#10;}</pre> | Caching options for the data
```

Comment:
It would help to specify what individual options change, as `limit` and `ttl` are somewhat overloaded. Perhaps something like:
```suggestion
cache | false | <pre>{&#13;&#10;&nbsp;&nbsp;limit: 5000,&#13;&#10;&nbsp;&nbsp;ttl: 300000&#13;&#10;}</pre> | Caching options for the data - `cache.limit` - the maximum number of records, and `cache.ttl` - initial time to hold a record in ms.
```

Reply:
Good point, though `ttl` is actually the of time to live for a record.

Reply:
Ah I see the library now uses LRU cache with a fixed TTL as opposed to starting with a small TTL that grows on every read. 👍